pkg/actors/placement/placement.go

```diff
@@ -532,7 +536,14 @@ func (p *actorPlacement) updatePlacements(in *v1pb.PlacementTables) {
 			for lk, lv := range v.GetLoadMap() {
 				loadMap[lk] = hashing.NewHost(lv.GetName(), lv.GetId(), lv.GetLoad(), lv.GetPort())
 			}
-			p.placementTables.Entries[k] = hashing.NewFromExisting(v.GetHosts(), v.GetSortedSet(), loadMap)
+
+			// TODO in v1.14 remove the check for versions < 1.13
```

Comment:
Probably 1.15, we try to keep n-2 compat

Reply:
Got it. Updating all instances of this note.